crates/pm_auth/src/types/api.rs

```diff
@@ -38,8 +38,8 @@ pub trait ConnectorIntegration<T, Req, Resp>: ConnectorIntegrationAny<T, Req, Re
     fn get_request_body(
         &self,
         _req: &super::PaymentAuthRouterData<T, Req, Resp>,
-    ) -> CustomResult<Option<RequestBody>, ConnectorError> {
-        Ok(None)
+    ) -> CustomResult<RequestContent, ConnectorError> {
+        Ok(RequestContent::Json(Box::new(serde_json::json!(r#""#))))
```

Comment:
```suggestion
        Ok(RequestContent::Json(Box::new(serde_json::json!(r#"{}"#))))
```
Since this method will return http request body, let's default it to empty json object instead of empty string